dash-renderer/src/components/error/FrontEnd/FrontEndError.react.js

```diff
@@ -140,6 +142,19 @@ function UnconnectedErrorContent({error, base}) {
                         />
                     </div>
                 </div>
+            ) : (
+                <div className="dash-be-error__str">
+                    <div
+                        className="dash-backend-error"
+                        style={{
+                            width: 'calc(600px - 67px)',
+                            height: '75vh',
+                            border: 'none',
```

Comment:
Do we actually need any of these style attrs?

Reply:
I kept them because we used them previously, but it might be ok to 🔪 from both R and Python. The important bits are in the `FrontEndError.css` file anyhow.

Reply:
The `<iframe>` needs explicit sizing because the containing page doesn't know what's in it, and border because perhaps iframes have a border by default. But the unframed string version should be closer to the front-end version, which gets most of its styling from `.dash-fe-error__info`.

Reply:
fixed in bf089447f31618973412356e690d011e911789c9